server/officers.lua

```diff
@@ -16,8 +16,8 @@ SetInterval(function()
 
     for _, officer in pairs(activeOfficers) do
         local coords = GetEntityCoords(officer.ped)
-        officer.position[1] = coords.y
-        officer.position[2] = coords.x
+        officer.position[1] = coords.x
+        officer.position[2] = coords.y
```

Comment:
Coords were reversed in [this](https://github.com/overextended/ox_mdt/pull/27) PR. Does this reverse them again?

Reply:
Not the same file, it was a mistake everyhere, and this is what we actually use on prod with the map update